docs/source/tutorial.md

```diff
@@ -0,0 +1,168 @@
+# Tutorial
+
+One of the main things that conda apart from other python packaging tool solutions is its ability to work with other languages and toolchains.  Conda
+is happy to mix Python with packages from R, Java, C, etc. This tutorial explores using **conda-project** to create a [Shiny](https://shiny.rstudio.com/)
+app using both **Jupyter notebook** on the Python side, and **shiny** on the R side.  After completing this tutorial, you will be able to:
+
+ * Understand the relationship between conda-project commands and their corresponding actions.
+ * Add a custom command to a project.
+ * Use the `run` action to launch external processes.
+
+If `conda-project` is not yet installed and started Project, follow the [Installation Instructions](index).
+
+## Create a new project
+
+This section of the tutorial explores the `create` and `activate` subcommands of conda-project.
+
+1. Open a Command Prompt or terminal window.
+
+2. Initialize the project in a new directory:
+   ```shell
+   conda-project create -n learn-cp --directory cp-tutorial --platform osx-arm64,linux-64 -c defaults -c conda-forge \
+       python=3.10 notebook r-essentials r-plotly r-shiny
+   ```
+
+   At this point, **conda-project** will create a new directory, and write environment, lock, and project files to the new
+   directory. This example passes the `osx-arm64` and `linux-64` platform architectures. This has the effect of speeding up
+   the package resolution and lock file generation.  If the `--platform` argument is omitted, package resolution will be
+   calculated across **conda-project**'s default list of platform architectures.
+
+   ```{note}
+   This tutorial issues the `create` action for a new project.  As noted in the [User Guide](user_guide), it is also possible
+   to create a project from an existing environment file.
+
+   If using windows, change the `--platform` argument to point to the correct architecture (win-64).
+   ```
+
+3. Navigate the newly created project directory:
+   ```shell
+   cd cp-tutorial
+   ```
+
+4. Investigate the contents of the newly created environment file:
+   ```shell
+   $ cat environment.yml
+   channels:
+     - defaults
+     - conda-forge
+   dependencies:
+     - python=3.10
+     - notebook
+     - r-essentials
+     - r-plotly
+     - r-shiny
+   platforms:
+     - osx-arm64
+     - linux-64
+   ```
+
+   ```{note}
+   Refer to [conda-project create](user_guide) for a detailed list of options.
+   ```
+
+5. Investigate the contents of the newly created conda project file:
+   ```yaml
+   $ cat conda-project.yml
+   name: learn-cp
+     environments:
+   default:
```

Comment:
`default:` doesn't appear to be at the correct indentation level.

Reply:
[cd3f3d](https://github.com/conda-incubator/conda-project/pull/72/commits/cd3f3d9914647f51fd2d45857fe0cf484e1a33ba)